src/components/Footer/Footer.module.css

```diff
@@ -36,8 +36,11 @@
 }
 
 .footerLink {
-    padding: 12px 32px;
-
+    padding: 12px;
+    @media (min-width: 375px) {
+        padding-left: 18px;
+        padding-right: 18px;
```

Comment:
Same thing as last time: these should be based on a global spacing set in config.

Reply:
don't think, we have to use global spacing set for current project. it would be just extra work here